frontend/www/js/omegaup/components/course/Administrators.vue

```diff
@@ -51,26 +48,22 @@
                 <td><button class="close"
                         type="button"
                         v-if="admin.role != 'site-admin' &amp;&amp; admin.role != 'owner'"
-                        v-on:click="onRemoveAdmin(admin)">×</button></td>
+                        v-on:click="$emit('removeAdmin', admin);">×</button></td>
```

Comment:
nit:

```suggestion
                        v-on:click="$emit('removeAdmin', admin)">×</button></td>
```

igual L96.